lib/Wikia/src/Service/Helios/HeliosClientImpl.php

```diff
@@ -123,57 +136,63 @@ protected function processResponseOutput( \MWHttpRequest $request ) {
 	/**
 	 * A shortcut method for login requests.
 	 *
-	 * @throws ClientException
+	 * @param $username
+	 * @param $password
+	 *
+	 * @return array
 	 */
-	public function login( $username, $password )
-	{
+	public function login( $username, $password ) {
 		// Convert the array to URL-encoded query string, so the Content-Type
 		// for the POST request is application/x-www-form-urlencoded.
 		// It would be multipart/form-data which is not supported
 		// by the Helios service.
-		$postData = http_build_query([
-			'username'	=> $username,
-			'password'	=> $password
-		]);
+		$postData = [
+			self::USERNAME => $username,
+			self::PASSWORD => $password,
+		];
 
 		$response = $this->request(
 			'token',
 			[],
 			$postData,
-			[ 'method'	=> 'POST' ]
+			[ self::METHOD => 'POST' ]
 		);
 
-		return [$this->status, $response];
+		return [ $this->status, $response ];
 	}
 
 	/**
 	 * A shortcut method to remove all tokens for user in helios
 	 *
 	 * @param $userId int for remove user tokens
+	 *
 	 * @internal param $username
 	 * @return null
 	 */
 	public function forceLogout( $userId ) {
 		return $this->request(
 			sprintf( 'users/%s/tokens', $userId ),
-			[ ],
-			[ ],
+			[],
+			[],
 			[
-				'method' => 'DELETE',
-				'headers' => [ self::SCHWARTZ_HEADER_NAME => $this->schwartzToken ]
+				self::METHOD  => 'DELETE',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "DELETE" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)